### PR TITLE
plat-stm32mp1: BSEC drivers fixes and updates

### PR DESCRIPTION
**Pull request by etienne-lms**

### CODE REVIEW

File: core/include/drivers/stm32_bsec.h [hunk 1]
```diff
@@ -10,6 +10,9 @@
 #include <stdint.h>
 #include <tee_api.h>
 
+#define BSEC_BITS_PER_WORD		(8U * sizeof(uint32_t))
+#define BSEC_BYTES_PER_WORD		(sizeof(uint32_t))
```
Comment (jforissier) [inline core/include/drivers/stm32_bsec.h:14]: This one is not used

**Review by jforissier — COMMENTED** [review]
- For commit "drivers: stm32_bsec: define BSEC word size in header file":
You do not say why you move the definition to the header file. Is this patch really useful? Which C file other than `stm32_bsec.h` (Edit: I mean stm32_bsec.**c**) would need this definition?
- For other commits:
`Acked-by: Jerome Forissier <jerome@forissier.org>`

**Review by etienne-lms — COMMENTED** [review]
> You do not say why you move the definition to the header file. Is this patch really useful? Which C file other than stm32_bsec.h would need this definition?

Well.. my apologies, dinging a bit, it's useless. I'll discard the change. Thanks for pointing that :), Further change are expected to provide more flexibility in OTPs identification. But actaully this change in even useless in that scope.
Further change will ease definition of OTPs using a nvmem layout, binding nvmem areas (OTP ranges) to identifiers (string names). This

**Comment by etienne-lms** [comment]
Removed commit "drivers: stm32_bsec: define BSEC word size in header file" from the series.
Applied tag to commits.

**Comment by etienne-lms** [comment]
Right, i'll fix the commit message.
Thanks

**Comment by etienne-lms** [comment]
Commit log fixed.
Tags applied.

**Pull request merged**